java/src/apps/CheckForUpdateAction.java

```diff
@@ -133,8 +123,9 @@ public void actionPerformed(ActionEvent event) {
         frame.setVisible(true);
     }
 
-    @Nonnull String getNumber(@Nonnull BufferedReader reader) throws java.io.IOException {
-        reader.readLine();  // skip a line
+    @Nonnull
+    String getNumber(@Nonnull BufferedReader reader) throws java.io.IOException {
+        log.debug("skipping line {}", reader.readLine()); // skip a line
```

Comment:
I tend to overlook logging statements when reading the code, so I missed this `readLine()` the first time through.  Would it be better to have it on a separate line?  To include the output in the `log.debug` would require a String on the stack, though.  No change needed, just a thought.

Reply:
To be fair, I changed this to stop NetBeans complaining about the result of readLine being ignored ( which is the intention ).
There's a likely a better way to achieve these than obsucing the code objective within debug statements, I'll do a bit of searching. . . .

Reply:
Would NB complain about:
```
   String skippedLine = reader.readLine();
   log.debug("skipping line {}", skippedLine);
```